src/connector.rs

```diff
@@ -1,6 +1,8 @@
 //! Utility methods for instantiating common connectors for clients.
 extern crate hyper_tls;
+#[cfg(not(target_os = "macos"))]
 extern crate native_tls;
+#[cfg(not(target_os = "macos"))]
```

Comment:
Should we be following the example of https://github.com/sfackler/rust-native-tls/blob/v0.1.5/src/backend/mod.rs#L9 here (and elsewhere) - i.e. `#[cfg(not(any(target_os = "macos", target_os = "windows", target_os = "ios")))]`

Reply:
done